aws/rds.md

```diff
@@ -0,0 +1,26 @@
+# AWS Relational Database Service (RDS)
+
+This doc presents guides on working with AWS RDS, Amazon's Database-as-a-Service provider.
+
+## Connecting with an EC2 instance
+
+In order to connect to an RDS instance from an EC2 instance, you'll need to configure the security rules for the database to allow access from the instance. Broadly speaking, AWS allows you to do this by 1) creating the database and the EC2 instance in the same Virtual Private Cloud (VPC) and 2) creating an inbound rule for the database's security group that permits access to PostgreSQL from the EC2 instance's security group.
```

Comment:
i like that you've been able distill all these steps down into two, high-level actions. how would you feel about using these two actions as headings, and placing the relevant steps in each section? for me, it's great to know the intent as i'm going through the actions.

Reply:
Good call!